woocommerce/woocommerce-android
# [Login] Jetpack connection for site credentials login Part 2

Part of #7307 

### Description
This PR adds the second part of handling the Jetpack Connection when signing using site credentials to a Jetpack site that's not connected to our account.

### Testing instructions

1. Prepare a site that has two accounts, where one of them is connected to Jetpack.
2. Open the app, and sign in using the site address.
3. Use site credentials for login, and enter credentials of the non-connected account.
4. Click on "Connect Jetpack to your account"
5. Continue the connection on the WebView.
6. Confirm the WebView is closed after finishing the connection.
7. Notice a loading indicator for fetching Jetpack User data.
8. After the loading is done, confirm that you are taken to the email verification screen in LoginActivity.


### Images/gif
https://user-images.githubusercontent.com/1657201/188929994-9e2fd423-a806-49ce-a9e9-813389e31c10.mov
